### PR TITLE
Fixing #2838

### PR DESCRIPTION
**Pull request by FinnStutzenstein**

### CODE REVIEW

File: openslides/motions/static/js/motions/site.js [hunk 1]
```diff
@@ -1112,9 +1112,16 @@ angular.module('OpenSlidesApp.motions.site', [
         Tag.bindAll({}, $scope, 'tags');
         User.bindAll({}, $scope, 'users');
         Workflow.bindAll({}, $scope, 'workflows');
-        MotionChangeRecommendation.bindAll({'where': {'motion_version_id': {'==': motion.active_version}}}, $scope, 'change_recommendations');
         Motion.loadRelations(motion, 'agenda_item');
         $scope.$watch(function () {
+            return MotionChangeRecommendation.lastModified();
+        }, function () {
+            $scope.change_recommendations = MotionChangeRecommendation.filter({'where': {'motion_version_id': {'==': motion.active_version}}});
```
Comment (normanjaeckel) [inline openslides/motions/static/js/motions/site.js]: THis line is too long.

Comment (FinnStutzenstein) [inline openslides/motions/static/js/motions/site.js]: @normanjaeckel Done.

**Comment by normanjaeckel** [comment]
Please rebase.

**Comment by FinnStutzenstein** [comment]
@normanjaeckel Rebased.

**Pull request merged**